reduction/imaging_parameters.py

```diff
@@ -3641,7 +3641,7 @@
     "G328.25_B6_12M_robust0_spw5": {
         "threshold": "5sigma",
         "scales": [0, 4, 8, 16, 32],
-        "startmodel": "G328.25_B6_uid___A001_X1296_X163_continuum_merged_12M_robust0_selfcal4_finaliter",
+        "startmodel": "G328.25_B6_uid___A001_X1296_X163_continuum_merged_12M_robust0_selfcal4_finaliter", 
```

Comment:
you added trailing whitespace here - that's not allowed, please re-run `black` following the command at the top of the script, then push the updated commit

Reply:
I thin I've done it now, can you check please ? Sorry, I'm a gitHub beginner... Thanks !

Reply:
yep, you got it, thanks.

Did you use python's `black` for this, or did you just delete the space?  Either way is fine, I just want to make sure you get practice with the tools we're using.

Reply:
ah, it looks like you just deleted the whitespace.  Commit ee91613 shows the effect of running `black`.

Reply:
Nope I just deleted it. I don't know how to use python's black to be honest !

Reply:
see https://github.com/Maxime-Valeille/reduction/blob/692be6763550cd27a4a0c8c1b69f9f20375dced1/reduction/imaging_parameters.py#L36-L39

Reply:
Okay now I see the command in the script, but sorry for the stupid question, how do I re-run it in the code ? I think i have to un-comment it, but then ? 
I will be able to do it for next times then... Thanks

Reply:
Okay I see. Next time I will download the code and run the command line if necessary. Thank you !